packs/classfeatures/the-silent-whisper.json

```
@@ -141,6 +141,18 @@
                         "title": "PF2E.TraitAmp"
                     }
                 ]
+            },
+            {
+                "key": "DamageDice",
+                "override": {
+                    "dieSize": "d10"
+                },
+                "predicate": [
+                    "amp-spell",
+                    "item:slug:daze",
+                    "item:tag:psi-cantrip"
```

```suggestion
                    "item:slug:daze",
                    "item:tag:amped"
```

Now you can do this!